Documentation/Relationships.md

```diff
@@ -0,0 +1,193 @@
+It's very common to have custom models that relate to other custom models.
+When all your models conform to `Decodable`, Argo makes it really easy to
+populate those relationships. Let's look at a `Post` and `Comment` model and
+how they relate.
+
+Our `Post` model will be very simple:
+
+```swift
+struct Post {
+  let author: String
+  let text: String
+}
+```
+
+Then, implementing `Decodable` for `Post` we get this:
+
+```swift
+extension Post: Decodable {
+  static func decode(j: JSON) -> Decoded<Post> {
+    return curry(self.init)
+      <^> j <| "author"
+      <*> j <| "text"
+  }
+}
+```
+
+And the json looks like this:
```

Comment:
Also, since this is an example, what do you think about "Encoding a Post could then produce the following JSON:"?